rust/kcl-lib/tests/crazy_multi_profile/ops.snap

```diff
@@ -148,6 +148,22 @@ description: Operations executed crazy_multi_profile.kcl
     "type": "UserDefinedFunctionReturn"
   },
   {
+    "type": "UserDefinedFunctionCall",
+    "name": "revolve",
```

Comment:
Revolve is still getting treated as a user-defined function.

Reply:
Working on it...